examples/syscall-steal.c

```diff
@@ -43,6 +43,14 @@
 
 #if defined(CONFIG_KPROBES)
 #define HAVE_KPROBES 1
+#if defined(CONFIG_X86_64)
+ /* For x86 architecture, the syscall table cannot be used to invoke a syscall
+ * after commit 1e3ad78.
```

Comment:
Adding the description of the kernel version, i.e., `... after v6.9 (commit 1e3ad78) ...`, would be better.

Reply:
This commit has been backported to long term stable kernels, like v5.15, v6.1, v6.6 and v6.8, which is added in comment for now.